mautic/mautic
# [BUGFIX] Removed apostrophe from email validation

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Issue(s) addressed                     | Fixes #10411




#### Description:
Changed the email-validation so now Mautic allows an apostrophe in the email when importing contacts. 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a CSV with one email row and one name row
3. Set the email to have an apostrophe in it. (test.o'neil@australia.com for example)
4. Import the CSV
5. There should be no error anymore and the contact should be created as expected
